src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/kafka/KafkaOppfolgingstilfellePersonServiceSpek.kt

```diff
@@ -116,17 +116,41 @@ class KafkaOppfolgingstilfellePersonServiceSpek : Spek({
             "key2",
             kafkaOppfolgingstilfellePersonNotDialogmotekandidat,
         )
-        val kafkaOppfolgingstilfellePersonDialogmotekandidatLast = generateKafkaOppfolgingstilfellePerson(
+        val kafkaOppfolgingstilfellePersonNotDialogmotekandidatBackDated = generateKafkaOppfolgingstilfellePerson(
             personIdentNumber = ARBEIDSTAKER_PERSONIDENTNUMBER,
-            oppfolgingstilfelleDurationInDays = DIALOGMOTEKANDIDAT_STOPPUNKT_DURATION_DAYS + 1
+            start = LocalDate.now().minusDays(110),
+            oppfolgingstilfelleDurationInDays = DIALOGMOTEKANDIDAT_STOPPUNKT_DURATION_DAYS - 1,
         )
-        val kafkaOppfolgingstilfellePersonDialogmotekandidatLastRecord = ConsumerRecord(
+        val kafkaOppfolgingstilfellePersonNotDialogmotekandidatBackDatedRecord = ConsumerRecord(
             OPPFOLGINGSTILFELLE_PERSON_TOPIC,
             partition,
             3,
             "key3",
+            kafkaOppfolgingstilfellePersonNotDialogmotekandidatBackDated,
+        )
+        val kafkaOppfolgingstilfellePersonDialogmotekandidatLast = generateKafkaOppfolgingstilfellePerson(
```

Comment:
Samme her, det er for mye informasjon på en gang, jeg vet ikke hva som er viktig. Er ikke et oppfølgingstilfelle alltid knyttet til en person? Det er kanskje et ord som kan sløyfes også?